src/Pact/Gas/Table.hs

```diff
@@ -39,6 +39,9 @@ data GasCostConfig = GasCostConfig
   , _gasCostConfig_sortFactor :: Gas
   , _gasCostConfig_distinctFactor :: Gas
   , _gasCostConfig_concatenationFactor :: Gas
+  , _gasCostConfig_textConcatenationFactorOffset :: Gas -- up-front cost of text concatenation
```

Comment:
very long names, maybe `_textConcatFactorOffset` ?

Reply:
I'm fine with long names when we need to be extremely explicit